manifests/manager/manager.yaml

```diff
@@ -15,8 +15,6 @@ spec:
       labels:
```

Comment:
Why do we add manager.yaml in the separate folder ? Is it better to add all components under `/base` ?

Reply:
I am ok with both ways. I can move it there.

Reply:
I had kept that folder as it has sample controller manager config which we may want to use in future for https://github.com/kubeflow/tf-operator/issues/1350